types/underscore/index.d.ts

```diff
@@ -5217,6 +5261,46 @@ declare namespace _ {
          */
         isUndefined(): boolean;
 
+        /**
+         * Returns true if the wrapped object is a Set.
+         * @returns True if the wrapped object is a Set, otherwise false.
+         */
+        isSet(): boolean;
```

Comment:
Why not `V is Set<any>`?

Reply:
Tsc gives error on that, I think type predicate in this way cannot be done.